src/loaders/agent-base.js

```diff
@@ -14,9 +13,6 @@ export class AgentBase {
 
   constructor (agentIdentifier = generateRandomHexString(16)) {
     this.agentIdentifier = agentIdentifier
-
-    // Assign the observation context to the event emitter, so it knows how to create observation contexts
-    this.ee = ee.get(agentIdentifier)
```

Comment:
this was removed because in the micro-agent tests, this is getting invoked before the `configure` call is made which throws errors

Reply:
What's the error btw?

Reply:
it tries to access the config getters/setters and they dont exist yet